liblwgeom/liblwgeom.h.in

```diff
@@ -1833,7 +1833,7 @@ extern size_t gserialized_from_lwgeom_size(const LWGEOM *geom);
 * will contain the size of the final output, which is useful for setting the PgSQL 
 * VARSIZE information.
 */
-extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, int is_geodetic, size_t *size);
+extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, size_t *size);
```

Comment:
I'd think it would be better to fix that, honouring the flag.
But I'd leave last word to Paul, as the geography architect.